.changeset/large-moons-speak.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/catalog-client': minor
```

Comment:
This PR [added a new required method to the exported `CatalogApi` interface](https://github.com/backstage/backstage/pull/21975/files#diff-47ce190784b257e830ecee68d94a53cda265056243d13ecfda310cc43723fe9c).  Anyone already implementing that interface in their code would therefore need to add that method to remain compliant with that interface.  Wouldn't that be a breaking change, thus requiring a `major` version bump per semver?